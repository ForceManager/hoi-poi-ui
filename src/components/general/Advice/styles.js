export default (theme) => ({
    root: {
        ...theme.typography.defaultText,
        borderRadius: '8px',
        display: 'flex',
        padding: '16px',
        '& .ReactCollapse--collapse': {
            width: '100%',
            minHeight: 20,
            overflow: 'hidden !important',
            transition: 'height 500ms',
            willChange: 'height',
        },
    },
    icon: {
        display: 'flex',
        marginRight: 10,
        marginTop: 1,
    },
    Text: {
        display: 'block',
        width: '100%',
    },

    // Types
    default: {
        backgroundColor: theme.colors.neutral100,
        '& $Text': {
            color: theme.colors.neutral700,
        },
    },
    error: {
        backgroundColor: theme.colors.red100,
        '& $Text': {
            color: theme.colors.red500,
        },
    },
    info: {
        backgroundColor: theme.colors.aqua100,
        '& $Text': {
            color: theme.colors.aqua700,
        },
    },
    success: {
        backgroundColor: theme.colors.green100,
        '& $Text': {
            color: theme.colors.green700,
        },
    },
    warning: {
        backgroundColor: theme.colors.yellow100,
        '& $Text': {
            color: theme.colors.yellow700,
        },
    },
    textContainer: {
        flex: 1,
        overflow: 'hidden',
    },
    collapseContainer: {
        display: 'flex',
        overflow: 'hidden',
        alignItems: 'flex-start',
    },
    dropdownIcon: {
        lineHeight: '0',
        transition: 'transform 0.3s ease-out',
        willChange: 'transform',
        transform: 'rotate(-180deg) translateY(1px)',
        cursor: 'pointer',
    },
    isOpened: {
        '& $dropdownIcon': {
            transform: 'rotate(0) translateY(1px)',
        },
        '& $Text': {
            whiteSpace: 'pre-wrap',
        },
    },
    withTitleContainer: {
        marginTop: 15,
    },
});
