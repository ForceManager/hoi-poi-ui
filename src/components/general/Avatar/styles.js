export default (theme) => ({
    root: {
        ...theme.typography.defaultText,
        color: theme.colors.neutral700,
        overflow: 'hidden',
        background: theme.colors.neutral400,
        position: 'relative',
        '& > img': {
            height: '100%',
            width: '100%',
            objectFit: 'cover',
            textAlign: 'center',
        },
    },
    round: {
        borderRadius: '50%',
        '& $icon': {
            width: '80%',
            height: '80%',
            padding: '10%',
        },
    },
    square: {
        borderRadius: 4,
        '& $icon': {
            width: '96%',
            height: '96%',
            padding: '2%',
        },
    },
    tiny: {
        height: 18,
        width: 18,
        '& $initials': {
            fontSize: 10,
        },
        '& $count': {
            transform: 'translate3d(40%, 40%, 0)',
        },
    },
    small: {
        height: 20,
        width: 20,
        '& $initials': {
            fontSize: 10,
        },
        '& $count': {
            transform: 'translate3d(40%, 40%, 0)',
        },
    },
    medium: {
        height: 25,
        width: 25,
    },
    large: {
        height: 30,
        width: 30,
        '& $initials': {
            fontSize: 16,
        },
    },
    big: {
        height: 32,
        width: 32,
        '& $initials': {
            fontSize: 18,
        },
    },
    huge: {
        height: 64,
        width: 64,
        '& $initials': {
            fontSize: 38,
        },
        '& $count': {
            transform: 'none',
        },
    },
    icon: {
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
    },
    initials: {
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        width: '100%',
        height: '100%',
        textTransform: 'uppercase',
    },
    multiAvatar: {
        overflow: 'visible',
    },
    multiAvatarInner: {
        display: 'grid',
        overflow: 'hidden',
        borderRadius: '50%',
    },
    multiAvatarItem: {
        '& img': {
            width: '100%',
            height: '100%',
            objectFit: 'cover',
            display: 'block',
        },
        '&:first-child': {
            gridColumn: 1,
            gridRow: '1 / span 2',
            aspectRatio: 0.5,
        },

        '&:nth-child(2)': {
            gridColumn: 2,
            gridRow: 1,
            aspectRatio: 1,
        },

        '&:nth-child(3)': {
            gridColumn: 2,
            gridRow: 2,
            aspectRatio: 1,
        },

        '&:nth-child(2):is(:last-child)': {
            gridColumn: 2,
            gridRow: '1 / span 2',
            aspectRatio: 0.5,
        },

        '&:only-child': {
            gridColumn: '1 / span 2',
            gridRow: '1 / span 2',
            aspectRatio: 1,
        },
    },
    count: {
        position: 'absolute',
        bottom: 0,
        right: 0,
        boxSizing: 'border-box',
        boxShadow: `0 0 0 2px ${theme.colors.neutralBase}`,
        backgroundColor: theme.colors.neutral400,
        transform: 'translate3d(25%, 25%, 0)',
    },
});
