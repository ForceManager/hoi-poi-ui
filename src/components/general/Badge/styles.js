export default (theme) => ({
    root: {
        ...theme.typography.smallText,
        lineHeight: '20px',
        color: theme.colors.text.black,
        borderRadius: '3px',
        display: 'inline-block',
        padding: '0 10px',
        minWidth: 70,
        textAlign: 'center',
    },
    Text: {
        lineHeight: '20px',
        fontWeight: 500,
        textTransform: 'uppercase',
    },
    // Types
    error: {
        backgroundColor: theme.colors.redLight,
        '& $Text': {
            color: theme.colors.text.red,
        },
    },
    info: {
        backgroundColor: theme.colors.greyLight,
        '& $Text': {
            color: theme.colors.text.greySoft,
        },
    },
    success: {
        backgroundColor: theme.colors.greenLight,
        '& $Text': {
            color: theme.colors.text.green,
        },
    },
    warn: {
        backgroundColor: theme.colors.yellowLight,
        '& $Text': {
            color: theme.colors.text.yellow,
        },
    },
});
