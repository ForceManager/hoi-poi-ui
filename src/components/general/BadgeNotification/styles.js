export default (theme) => ({
    root: {
        display: 'inline-block',
        textAlign: 'center',
        backgroundColor: theme.colors.orange500,
        position: 'relative',
    },
    Text: {
        padding: '0px 8px',
        verticalAlign: 'middle',
        color: theme.colors.neutralBase,
    },
    medium: {
        height: 24,
        minWidth: 24,
        borderRadius: 24,
        '& $Text': {
            lineHeight: '24px',
        },
    },
    small: {
        height: 18,
        minWidth: 18,
        borderRadius: 18,
        '& $Text': {
            lineHeight: '18px',
        },
    },
});
