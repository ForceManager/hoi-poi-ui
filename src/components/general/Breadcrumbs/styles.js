export default (theme) => ({
    root: {
        display: 'flex',
        alignItems: 'center',
    },
    item: {
        marginRight: 8,
        '& span': {
            color: theme.colors.textLight.secondary,
        },
        '&:last-child': {
            margin: 0,
            '& span': {
                color: theme.colors.textLight.primary,
            },
        },
    },
    clickables: {
        '& $item': {
            cursor: 'pointer',
            '&:hover': {
                '& span': {
                    color: theme.colors.blue500,
                },
                '&:last-child': {
                    '& span': {
                        color: theme.colors.textLight.primary,
                    },
                },
            },
            '&:last-child': {
                cursor: 'inherit',
            },
        },
        '& $divider': {
            cursor: 'inherit',
        },
    },
    divider: {},
});
