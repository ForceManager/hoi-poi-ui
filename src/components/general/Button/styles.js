export default (theme) => ({
    root: {
        position: 'relative',
        boxSizing: 'border-box',
        display: 'inline-block',
        textAlign: 'center',
        borderRadius: 4,
        border: 'none',
        outline: 'none',
        cursor: 'pointer !important',
        '&$fullWidth': {
            width: '100%',
        },
        '&$disabled, &$loading': {
            opacity: 0.5,
            cursor: 'not-allowed !important',
            '&:active': {
                boxShadow: 'none',
            },
            '& *': {
                cursor: 'not-allowed',
            },
        },
        '& *': {
            cursor: 'pointer',
        },
    },
    content: {
        display: 'flex',
        alignItems: 'center',
        justifyContent: 'center',
    },
    Text: {
        display: 'inherit',
        alignItems: 'inherit',
        justifyContent: 'inherit',
        transition: 'opacity .2s ease-in-out',
        ...theme.typography.buttonText,
        lineHeight: '16px', // Ver typography
        color: 'inherit',
        fontSize: 'inherit',
    },
    loaderContainer: {
        position: 'absolute',
        display: 'flex',
        width: '100%',
        height: '100%',
        top: 0,
        left: 0,
        justifyContent: 'center',
        alignItems: 'center',
        '& div': {
            marginBottom: -0,
        },
    },
    icon: {},
    // labelLoading: {
    //     top: 1,
    //     position: 'relative',
    //     '& $Text': {
    //         opacity: 0,
    //     },
    //     '& $loader': {
    //         opacity: 1,
    //     },
    // },
    // loader: {
    //     position: 'absolute',
    //     top: 0,
    //     left: 0,
    //     width: '100%',
    //     height: '100%',
    //     display: 'flex',
    //     justifyContent: 'center',
    //     alignItems: 'center',
    //     opacity: 0,
    //     transition: 'opacity .2s ease-in-out',
    // },
    // Types
    primary: {
        backgroundColor: theme.colors.orange500,
        color: theme.colors.neutralBase,
        '&:not($loading):not($disabled)': {
            '&:hover': {
                backgroundColor: theme.colors.orange400,
            },
            '&:active': {
                backgroundColor: theme.colors.orange600,
            },
        },
    },
    primaryError: {
        backgroundColor: theme.colors.red500,
        color: theme.colors.neutralBase,
        '&:not($loading):not($disabled)': {
            '&:hover': {
                backgroundColor: theme.colors.red400,
            },
            '&:active': {
                backgroundColor: theme.colors.red600,
            },
        },
    },
    secondary: {
        border: `1px solid ${theme.colors.orange500}`,
        backgroundColor: theme.colors.neutralBase,
        color: theme.colors.orange500,
        '&:not($loading):not($disabled)': {
            '&:hover': {
                borderWidth: 2,
                color: theme.colors.orange400,
            },
            '&:active': {
                border: `2px solid ${theme.colors.orange600}`,
                backgroundColor: theme.colors.orange600,
                color: theme.colors.neutralBase,
                '& path': { fill: theme.colors.neutralBase },
            },
        },
    },
    secondaryError: {
        border: `1px solid ${theme.colors.neutral400}`,
        backgroundColor: theme.colors.neutralBase,
        color: theme.colors.red500,
        '&:not($loading):not($disabled)': {
            '&:hover': {
                border: `2px solid ${theme.colors.red400}`,
                color: theme.colors.red400,
            },
            '&:active': {
                border: `2px solid ${theme.colors.red600}`,
                backgroundColor: theme.colors.red600,
                color: theme.colors.neutralBase,
                '& path': { fill: theme.colors.neutralBase },
            },
        },
    },
    terciary: {
        border: `1px solid ${theme.colors.neutral400}`,
        backgroundColor: theme.colors.neutralBase,
        color: theme.colors.neutral700,
        '&:not($loading):not($disabled)': {
            '&:hover': {
                border: `2px solid ${theme.colors.neutral200}`,
                color: theme.colors.neutral600,
            },
            '&:active': {
                border: `2px solid ${theme.colors.neutral300}`,
                backgroundColor: theme.colors.neutral300,
                color: theme.colors.neutral800,
                '& path': { fill: theme.colors.neutral800 },
            },
        },
    },
    outline: {
        '&:not($loading):not($disabled)': {
            '&:hover, &:active': {
                '&$big': {
                    padding: '0 16px',
                },
                '&$medium': {
                    padding: '0 15px',
                },
                '&$small': {
                    padding: '0 9px',
                },
            },
        },
    },

    // Sizes
    big: {
        height: 44,
        padding: '0 17px',
        // Ver typography
        fontSize: 14,
    },
    medium: {
        height: 32,
        padding: '0 16px',
        // Ver typography
        fontSize: 14,
    },
    small: {
        height: 24,
        padding: '0 8px',
        // Ver typography
        fontSize: 12,
    },

    // States
    disabled: {},
    loading: {
        '& $Text': {
            opacity: 0,
        },
    },
    fullWidth: {},
    iconLeft: {
        '& $icon': {
            marginRight: 6,
        },
    },
    iconRight: {
        '& $icon': {
            marginLeft: 6,
        },
    },
    // outlined: {
    //     padding: '4px 20px',
    //     '&$primary': {
    //         backgroundColor: 'transparent',
    //         border: `1px solid ${theme.colors.primary}`,
    //         transition: `color .2s ease-in-out, box-shadow .2s ease-in-out, background-color .1s ease-in-out`,
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: 'transparent',
    //                 '& $Text': {
    //                     color: theme.colors.primary,
    //                 },
    //             },
    //         },
    //         '& $Text': {
    //             color: theme.colors.primary,
    //         },
    //         '&:hover': {
    //             backgroundColor: theme.colors.primarySoft,
    //             '& $Text': {
    //                 color: theme.colors.text.primary,
    //             },
    //         },
    //     },
    //     '&$danger': {
    //         background: 'transparent',
    //         backgroundColor: 'transparent',
    //         border: `1px solid ${theme.colors.red}`,
    //         transition: `color .2s ease-in-out, box-shadow .2s ease-in-out, background-color .1s ease-in-out`,
    //         '& $Text': {
    //             color: theme.colors.red,
    //         },
    //         '&:hover': {
    //             backgroundColor: theme.colors.red,
    //             '& $Text': {
    //                 color: theme.colors.text.white,
    //             },
    //         },
    //     },
    //     '&$small': {
    //         padding: '0 20px',
    //         '& $Text': {
    //             lineHeight: '18px',
    //         },
    //     },
    //     '&$big': {
    //         padding: '9px 20px',
    //     },
    // },
    // squared: {
    //     borderRadius: 2,
    //     padding: '4px 10px',
    //     '&$white': {
    //         backgroundColor: 'transparent',
    //         transition: `box-shadow .2s ease-in, background-color .1s ease-in-out`,
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: 'transparent !important',
    //             },
    //         },
    //         '& $Text': {
    //             color: theme.colors.text.default,
    //         },
    //         '&:hover': {
    //             backgroundColor: theme.colors.greyLight,
    //         },
    //     },
    //     '&$primary': {
    //         backgroundColor: theme.colors.primary,
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: `${theme.colors.primary} !important`,
    //             },
    //         },
    //         '&:hover': {
    //             backgroundColor: theme.colors.primarySoft,
    //         },
    //         '& $Text': {
    //             color: theme.colors.text.white,
    //         },
    //     },
    //     '&$danger': {
    //         backgroundColor: theme.colors.red,
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: `${theme.colors.red} !important`,
    //             },
    //         },
    //         '&:hover': {
    //             backgroundColor: theme.colors.redsoft,
    //         },
    //         '& $Text': {
    //             color: theme.colors.text.white,
    //         },
    //     },
    //     '&$small': {
    //         padding: '0 9px',
    //     },
    //     '&$big': {
    //         padding: '9px 9px',
    //     },
    // },
    // 'squared-outlined': {
    //     borderRadius: 2,
    //     padding: '4px 10px',
    //     backgroundColor: 'transparent',
    //     border: `1px solid ${theme.colors.placeholders}`,
    //     transition: `border-color .2s ease-in-out`,
    //     '&:hover': {
    //         border: `1px solid ${theme.colors.placeholdersHover}`,
    //         backgroundColor: 'transparent !important',
    //     },
    //     '&$disabled': {
    //         '&:hover': {
    //             border: `1px solid ${theme.colors.placeholders}`,
    //             backgroundColor: 'transparent',
    //         },
    //     },
    //     '&$white': {
    //         backgroundColor: 'transparent',
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: 'transparent',
    //             },
    //         },
    //         '&:hover': {
    //             backgroundColor: 'transparent',
    //         },
    //     },
    //     '&$primary': {
    //         backgroundColor: 'transparent',
    //         '&$disabled': {
    //             '&:hover': {
    //                 backgroundColor: 'transparent',
    //                 '& $Text': {
    //                     color: theme.colors.primary,
    //                 },
    //             },
    //         },
    //         '& $Text': {
    //             color: theme.colors.primary,
    //         },
    //     },
    //     '&$danger': {
    //         background: 'transparent',
    //         backgroundColor: 'transparent',
    //         '& $Text': {
    //             color: theme.colors.red,
    //         },
    //     },
    //     '&$small': {
    //         padding: '0 9px',
    //     },
    //     '&$big': {
    //         padding: '9px 9px',
    //     },
    // },
    // filled: {},
    // //Colors
    // white: {
    //     backgroundColor: 'transparent',
    //     transition: `box-shadow .2s ease-in, background-color .1s ease-in-out`,
    //     '& $Text': {
    //         color: theme.colors.text.default,
    //     },
    //     '&$disabled': {
    //         '&:hover': {
    //             backgroundColor: 'transparent',
    //         },
    //     },
    //     '&:hover': {
    //         backgroundColor: theme.colors.greyLight,
    //     },
    // },
    // primary: {
    //     backgroundColor: theme.colors.primary,
    //     transition: `box-shadow .2s ease-in, background-color .1s ease-in-out`,
    //     '& $Text': {
    //         color: theme.colors.text.primary,
    //     },
    //     '&$disabled': {
    //         '&:hover': {
    //             backgroundColor: theme.colors.primary,
    //         },
    //     },
    //     '&:hover': {
    //         backgroundColor: theme.colors.primarySoft,
    //     },
    // },
    // danger: {
    //     backgroundColor: theme.colors.red,
    //     transition: `box-shadow .2s ease-in, background-color .1s ease-in-out`,
    //     '& $Text': {
    //         color: theme.colors.text.white,
    //     },
    //     '&$disabled': {
    //         '&:hover': {
    //             backgroundColor: theme.colors.red,
    //         },
    //     },
    //     '&:hover': {
    //         backgroundColor: theme.colors.redSoft,
    //     },
    // },
    // grey: {
    //     backgroundColor: theme.colors.tableHover,
    //     transition: `box-shadow .2s ease-in, background-color .1s ease-in-out`,
    //     '& $Text': {
    //         color: theme.colors.text.default,
    //     },
    //     '&$disabled': {
    //         '&:hover': {
    //             backgroundColor: theme.colors.tableHover,
    //         },
    //     },
    //     '&:hover': {
    //         backgroundColor: theme.colors.tableHeader,
    //     },
    // },
});
