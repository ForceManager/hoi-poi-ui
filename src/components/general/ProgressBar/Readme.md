Default:

```jsx
<ProgressBar percent={10} />

<ProgressBar percent={20} />

<ProgressBar percent={60} />

<ProgressBar percent={100} />

```

### Component tree

---

-   https://github.com/react-component/progress
