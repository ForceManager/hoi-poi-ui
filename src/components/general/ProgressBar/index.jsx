import React from 'react';
import PropTypes from 'prop-types';

import { useTheme } from '../../../utils/styles';

import { Circle, Line } from 'rc-progress';

function ProgressBar({ type, ...props }) {
    const theme = useTheme();
    const Component = type === 'circle' ? Circle : Line;
    return (
        <Component
            prefixCls="hoi-poi-ui"
            strokeColor={theme.colors.primary}
            trailColor={theme.colors.placeholders}
            {...props}
        />
    );
}

ProgressBar.defaultProps = {
    type: 'line',
    percent: 0,
    prefixCls: 'rc-progress',
    strokeLinecap: 'round',
    strokeWidth: 0.555555555555556,
};

const mixedType = PropTypes.oneOfType([PropTypes.number, PropTypes.string]);

ProgressBar.propTypes = {
    className: PropTypes.string,
    type: PropTypes.oneOf(['line', 'circle']),
    percent: PropTypes.oneOfType([mixedType, PropTypes.arrayOf(mixedType)]),
    strokeColor: PropTypes.oneOfType([
        PropTypes.string,
        PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.object])),
        PropTypes.object,
    ]),
    strokeWidth: mixedType,
    style: PropTypes.object,
    trailColor: PropTypes.string,
    trailWidth: mixedType,
};

export default React.memo(ProgressBar);
