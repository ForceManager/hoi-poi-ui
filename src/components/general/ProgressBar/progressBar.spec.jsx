import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { HoiPoiProvider } from '../../../utils/styles';
import ProgressBar from '../../general/ProgressBar';

describe('ProgressBar', () => {
    test('with different sizes', () => {
        const wrapper = mount(
            <HoiPoiProvider>
                <ProgressBar percent={10} />
                <ProgressBar percent={20} />
                <ProgressBar percent={30} />
                <ProgressBar percent={50} />
            </HoiPoiProvider>,
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
