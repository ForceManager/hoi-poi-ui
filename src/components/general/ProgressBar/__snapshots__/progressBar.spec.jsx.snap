// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar with different sizes 1`] = `
<Component>
  <ThemeProvider
    theme={
      Object {
        "colors": Object {
          "black": "#262B34",
          "blackLight": "#5f656b",
          "blackberry": "#2e2d4c",
          "blackberryDark": "#24243C",
          "disabledGrey": "#ced4d9",
          "green": "#86AD20",
          "greenLight": "#e7f1d5",
          "grey": "#697782",
          "greyLight": "#f0f3f5",
          "greySoft": "#788590",
          "lines": "#dce3eb",
          "link": "#216db3",
          "linkDark": "#174a79",
          "overlay": "rgba(33,33,33,0.55)",
          "placeholders": "#e1e4e7",
          "placeholdersHover": "#d0d4d8",
          "primary": "#FF8C00",
          "primaryDark": "#C87600",
          "primaryLight": "#fef4e6",
          "primarySoft": "#ff9d26",
          "purple": "#AA5B9C",
          "purpleLight": "#EEDEEB",
          "red": "#f04540",
          "redDark": "#C43833",
          "redLight": "#FAE7E7",
          "redSoft": "#FF544F",
          "secondary": "#00A8Be",
          "secondaryGrey": "#e4eeff",
          "secondaryLight": "#D6EEF2",
          "secondaryOverlay": "rgba(2,168,189,0.10)",
          "secondarySoft": "#AADFE7",
          "shadow": "rgba(0, 0, 0, 0.25)",
          "tableHeader": "#f6f7f9",
          "tableHeaderHover": "#EBECEF",
          "tableHover": "#FAFBFC",
          "text": Object {
            "black": "#3c4248",
            "danger": "white",
            "default": "#697782",
            "green": "#86AD20",
            "greySoft": "#788590",
            "primary": "white",
            "red": "#f04540",
            "secondary": "#00A8Be",
            "white": "#FFFFFF",
            "yellow": "#E9B200",
          },
          "transparentBlack": "rgba(38, 43, 52, 0.8)",
          "white": "#FFFFFF",
          "yellow": "#E9B200",
          "yellowDark": "#BE9100",
          "yellowLight": "#FCF7E5",
        },
        "effects": Object {
          "boxShadow": "0 3px 9px 0 rgba(0, 0, 0, 0.25)",
          "buttonActive": Object {
            "boxShadow": "inset 0 -2px 4px 0 rgba(51, 51, 51, 0.23)",
          },
        },
        "typography": Object {
          "bigText": Object {
            "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": 20,
            "letterSpacing": 0.5,
          },
          "button": Object {
            "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": 11,
            "lineHeight": 1.5,
          },
          "defaultText": Object {
            "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": 13,
            "letterSpacing": 0.5,
          },
          "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
          "fontSizeBase": 13,
          "fontSizeSmall": 11,
          "largeText": Object {
            "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": 16,
            "letterSpacing": 0.5,
          },
          "letterSpacing": 0.5,
          "lineHeightBase": 1.5,
          "smallText": Object {
            "fontFamily": "\\"Roboto\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": 11,
            "letterSpacing": 0.5,
          },
        },
        "utils": Object {
          "scrollbar": Object {
            "&::-webkit-scrollbar": Object {
              "height": 17,
              "width": 17,
            },
            "&::-webkit-scrollbar-thumb": Object {
              "background": "#e1e4e7",
              "backgroundClip": "content-box",
              "border": "4px solid rgba(0, 0, 0, 0)",
              "borderRadius": 10,
            },
            "&::-webkit-scrollbar-thumb:window-inactive": Object {
              "background": "#e1e4e7",
              "backgroundClip": "content-box",
              "border": "4px solid rgba(0, 0, 0, 0)",
              "borderRadius": 10,
            },
          },
        },
      }
    }
  >
    <JssProvider
      classNamePrefix="HoiPoi__"
      generateId={[Function]}
      jss={
        Jss {
          "generateId": [Function],
          "id": 3,
          "options": Object {
            "Renderer": [Function],
            "createGenerateId": [Function],
            "id": Object {
              "minify": false,
            },
            "plugins": Array [],
          },
          "plugins": PluginsRegistry {
            "plugins": Object {
              "external": Array [
                Object {
                  "onCreateRule": [Function],
                  "onProcessStyle": [Function],
                  "onUpdate": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                  "onProcessRule": [Function],
                },
                Object {
                  "onProcessRule": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                  "onProcessRule": [Function],
                },
                Object {
                  "onChangeValue": [Function],
                  "onProcessStyle": [Function],
                },
                Object {
                  "onProcessStyle": [Function],
                },
                Object {
                  "onProcessStyle": [Function],
                },
                Object {
                  "onChangeValue": [Function],
                  "onProcessStyle": [Function],
                },
                Object {
                  "onChangeValue": [Function],
                  "onProcessStyle": [Function],
                },
                Object {
                  "onProcessStyle": [Function],
                },
                Object {
                  "onChangeValue": [Function],
                  "onProcessRule": [Function],
                  "onProcessStyle": [Function],
                },
                Object {
                  "onProcessStyle": [Function],
                },
              ],
              "internal": Array [
                Object {
                  "onCreateRule": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                },
                Object {
                  "onChangeValue": [Function],
                  "onCreateRule": [Function],
                  "onProcessStyle": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                },
                Object {
                  "onCreateRule": [Function],
                },
              ],
            },
            "registry": Object {
              "onChangeValue": Array [
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
              ],
              "onCreateRule": Array [
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
              ],
              "onProcessRule": Array [
                [Function],
                [Function],
                [Function],
                [Function],
              ],
              "onProcessSheet": Array [],
              "onProcessStyle": Array [
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
                [Function],
              ],
              "onUpdate": Array [
                [Function],
              ],
            },
          },
          "version": "10.3.0",
        }
      }
    >
      <ProgressBar
        percent={10}
        prefixCls="rc-progress"
        strokeLinecap="round"
        strokeWidth={0.555555555555556}
        type="line"
      >
        <Line
          className=""
          percent={10}
          prefixCls="rc-progress"
          strokeColor="#FF8C00"
          strokeLinecap="round"
          strokeWidth={0.555555555555556}
          style={Object {}}
          trailColor="#e1e4e7"
          trailWidth={1}
        >
          <svg
            className="rc-progress-line"
            preserveAspectRatio="none"
            style={Object {}}
            viewBox="0 0 100 0.555555555555556"
          >
            <path
              className="rc-progress-line-trail"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              stroke="#e1e4e7"
              strokeLinecap="round"
              strokeWidth={1}
            />
            <path
              className="rc-progress-line-path"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              key="0"
              stroke="#FF8C00"
              strokeLinecap="round"
              strokeWidth={0.555555555555556}
              style={
                Object {
                  "strokeDasharray": "10px, 100px",
                  "strokeDashoffset": "-0px",
                  "transition": "stroke-dashoffset 0.3s ease 0s, stroke-dasharray .3s ease 0s, stroke 0.3s linear",
                }
              }
            />
          </svg>
        </Line>
      </ProgressBar>
      <ProgressBar
        percent={20}
        prefixCls="rc-progress"
        strokeLinecap="round"
        strokeWidth={0.555555555555556}
        type="line"
      >
        <Line
          className=""
          percent={20}
          prefixCls="rc-progress"
          strokeColor="#FF8C00"
          strokeLinecap="round"
          strokeWidth={0.555555555555556}
          style={Object {}}
          trailColor="#e1e4e7"
          trailWidth={1}
        >
          <svg
            className="rc-progress-line"
            preserveAspectRatio="none"
            style={Object {}}
            viewBox="0 0 100 0.555555555555556"
          >
            <path
              className="rc-progress-line-trail"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              stroke="#e1e4e7"
              strokeLinecap="round"
              strokeWidth={1}
            />
            <path
              className="rc-progress-line-path"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              key="0"
              stroke="#FF8C00"
              strokeLinecap="round"
              strokeWidth={0.555555555555556}
              style={
                Object {
                  "strokeDasharray": "20px, 100px",
                  "strokeDashoffset": "-0px",
                  "transition": "stroke-dashoffset 0.3s ease 0s, stroke-dasharray .3s ease 0s, stroke 0.3s linear",
                }
              }
            />
          </svg>
        </Line>
      </ProgressBar>
      <ProgressBar
        percent={30}
        prefixCls="rc-progress"
        strokeLinecap="round"
        strokeWidth={0.555555555555556}
        type="line"
      >
        <Line
          className=""
          percent={30}
          prefixCls="rc-progress"
          strokeColor="#FF8C00"
          strokeLinecap="round"
          strokeWidth={0.555555555555556}
          style={Object {}}
          trailColor="#e1e4e7"
          trailWidth={1}
        >
          <svg
            className="rc-progress-line"
            preserveAspectRatio="none"
            style={Object {}}
            viewBox="0 0 100 0.555555555555556"
          >
            <path
              className="rc-progress-line-trail"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              stroke="#e1e4e7"
              strokeLinecap="round"
              strokeWidth={1}
            />
            <path
              className="rc-progress-line-path"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              key="0"
              stroke="#FF8C00"
              strokeLinecap="round"
              strokeWidth={0.555555555555556}
              style={
                Object {
                  "strokeDasharray": "30px, 100px",
                  "strokeDashoffset": "-0px",
                  "transition": "stroke-dashoffset 0.3s ease 0s, stroke-dasharray .3s ease 0s, stroke 0.3s linear",
                }
              }
            />
          </svg>
        </Line>
      </ProgressBar>
      <ProgressBar
        percent={50}
        prefixCls="rc-progress"
        strokeLinecap="round"
        strokeWidth={0.555555555555556}
        type="line"
      >
        <Line
          className=""
          percent={50}
          prefixCls="rc-progress"
          strokeColor="#FF8C00"
          strokeLinecap="round"
          strokeWidth={0.555555555555556}
          style={Object {}}
          trailColor="#e1e4e7"
          trailWidth={1}
        >
          <svg
            className="rc-progress-line"
            preserveAspectRatio="none"
            style={Object {}}
            viewBox="0 0 100 0.555555555555556"
          >
            <path
              className="rc-progress-line-trail"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              stroke="#e1e4e7"
              strokeLinecap="round"
              strokeWidth={1}
            />
            <path
              className="rc-progress-line-path"
              d="M 0.277777777777778,0.277777777777778
         L 99.72222222222223,0.277777777777778"
              fillOpacity="0"
              key="0"
              stroke="#FF8C00"
              strokeLinecap="round"
              strokeWidth={0.555555555555556}
              style={
                Object {
                  "strokeDasharray": "50px, 100px",
                  "strokeDashoffset": "-0px",
                  "transition": "stroke-dashoffset 0.3s ease 0s, stroke-dasharray .3s ease 0s, stroke 0.3s linear",
                }
              }
            />
          </svg>
        </Line>
      </ProgressBar>
    </JssProvider>
  </ThemeProvider>
</Component>
`;
